tsconfig.json

```diff
@@ -39,7 +39,7 @@
   },
   "include": ["./types", "./src"],
   "exclude": [
-    "src/analysis/retail/deathknight/frost",
+    //"src/analysis/retail/deathknight/frost",
```

Comment:
Delete the line from the excludes rather than comment it out so its clear that we want to include the module and are not temporarily removing it.